.github/workflows/linux.yml

```diff
@@ -882,6 +921,7 @@ jobs:
         shell: bash
     runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-8-cores' || 'ubuntu-20.04'}}
     env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
```

Comment:
do we need it here?
dependencies are not installed in this job

Reply:
I do not undestand - why was it added for `PyTorch_Models_Tests` but not for `TensorFlow_Hub_Models_Tests`?
May be you missed:)

Reply:
it's not required in both cases, because we don't install apt packages in these jobs